dbms/src/Common/Exception.cpp

```diff
@@ -23,8 +23,21 @@ namespace ErrorCodes
 
 void throwFromErrno(const std::string & s, int code, int e)
 {
+#ifndef _GNU_SOURCE
+	const size_t bufSize = 256;
+	const char *unknownMessage = "Unknown error ";
+	char buf[bufSize];
```

Comment:
But there is discrepancy with code below,
`char buf[128]`
both in size of buffer and in method of specification of constant.